WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift

```diff
@@ -8,10 +8,23 @@ import AuthenticationServices
 ///
 @available(iOS 13, *)
 class StoredCredentialsAuthenticator: NSObject {
+
+    // MARK: - Delegates
+    
+    private var authenticationDelegate: WordPressAuthenticatorDelegate {
+        guard let delegate = WordPressAuthenticator.shared.delegate else {
+            fatalError()
```

Comment:
Is there a more graceful way to fail when the delegate doesn't exist?

Reply:
This is a bit ugly I agree, but I don't feel the solution is making this optional as this delegate isn't supposed to be absent.

In addition to this, I opted (fow now) to use the same snippet we're using in the [Apple Authenticator](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/8318b4183b7cbc016252b94ab291cf7e3bdef5f3/WordPressAuthenticator/Signin/AppleAuthenticator.swift#L29-L34) and the [Google Authenticator](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/8318b4183b7cbc016252b94ab291cf7e3bdef5f3/WordPressAuthenticator/Unified%20Auth/GoogleAuthenticator.swift#L107-L112).

With all that said, I'm planning to make this better in an upcoming PR in order to avoid scope creep in this PR.

Reply:
Oh that's fair. If that's how we usually handle missing delegates, then ignore that request.